pallets/creator-staking/src/inflation.rs

```diff
@@ -0,0 +1,44 @@
+use crate::*;
+use frame_support::traits::{Currency, Get, OnTimestampSet, Imbalance};
+use sp_runtime::traits::{Saturating, SaturatedConversion, UniqueSaturatedInto, Zero};
+
+type NegativeImbalanceOf<T> = <<T as Config>::Currency as Currency<
+    <T as frame_system::Config>::AccountId,
+>>::NegativeImbalance;
+
+impl<Moment, T: Config> OnTimestampSet<Moment> for Pallet<T> {
+    fn on_timestamp_set(_moment: Moment) {
+        let total_issuance = T::Currency::total_issuance();
+        let reward_amount: BalanceOf<T> = T::CurrentAnnualInflation::get() * total_issuance
+            / T::BlocksPerYear::get().saturated_into::<u32>().unique_saturated_into();
```

Comment:
Let's extract this to fn, for a better readability. For example: `fn calc_per_block_rewards()`